src/quantum/README.rst

```diff
@@ -0,0 +1,188 @@
+=======================================
+Microsoft Azure CLI 'quantum' Extension
+=======================================
+
+Azure Quantum is the first open Quantum computing platform. It offers a range of services
+from quantum hardware to full-state simulators and quantum inspired optimizations,
+providing developers and customers access to the most competitive quantum offering
+on the market.
+
+To learn more about azure quantum, visit:
+https://azure.microsoft.com/en-us/services/quantum/
+
+To learn more about quantum computing and Microsoft's Quantum Development Kit, visit:
+https://docs.microsoft.com/quantum/
+
+
+Using the `az quantum` extension to list and manage jobs in Azure Quantum
+=========================================================================
+
+1. Log in to Azure using your credentials.
+
+   .. code-block:: 
+
+      az login
+
+   .. note::
+      In case you have more than one subscription associated with your Azure account you must specify the 
+      subscription you want to use. You can do this with the command `az account set -s <Your subscription ID>`.
+
+
+2. Install the Quantum extension for the Azure CLI
+
+   .. code-block::
+
+      az extension add --name quantum
+
+
+3. You can see all the Azure Quantum workspaces in your subscription with the `az quantum workspace list` command:
+
+   .. code-block::
+
+      az quantum workspace list
+
+
+4. You can use `quantum workspace set` to select a default workspace you want to use 
+   to list and submit jobs. Note that you also need to specify the resource group.
+
+   .. code-block::
+
+      az quantum workspace set -g MyResourceGroup -w MyWorkspace  -o table
+
+      Location     Name                               ResourceGroup
+      -----------  ---------------------------------  --------------------------------
+      westus       ws-yyyyyy                          rg-yyyyyyyyy
+
+
+5. You can check the current default workspace with command `az quantum workspace show`.
+
+   .. code-block::
+
+      az quantum workspace show -o table
```

Comment:
ValidationError: Missing resource-group argument

Reply:
If you have previously set a default workspace (as in step #4 above, you don't need to provide resource group, workspace name or location, since the CLI will know). I'll add a clarification in step 4 itself to fix that.

Reply:
Got it